portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,59 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="author" content="Essence Cain">
+    <meta name="description" content="Essence Cain's portfolio page">
+    <meta name="keywords" content="Essence Cain, portfolio, NCAT">
+    <title>Essence Cain's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hello!</h1>
+      <h2>I am Essence Cain</h2>
+
+      <h3>About Me</h3>
+      <p>I am a current Computer Science student at North Carolina A&T State University, where I have began to explore mobile development and UX/UI design.</p>
+      <p>I believe technology will have a strong impact in advancing our society and I want to play a key role by offering a different perspective from undereppresnted groups 
+      in STEM fields.</p>
+
+      <h3>Work Experience</h3>
+      <img src="/images/google_icon.jpeg" alt="Google Logo" style="height:60px;width:60px">      
```

Comment:
Adding styling in the stylesheet is preferred over inline styling. That way, it keeps the structure and styling concerns separate and makes it easier for things like applying different styling to the same template. You can add a class name to this element and move the inline style to the css file. (same for the other images before).